electronics/ohms_law.py

```diff
@@ -0,0 +1,32 @@
+# Importing doctest for testing our function
+from doctest import testmod
+
+# Defining function
+
+
+def ohms_law(voltage: float = 0, current: float = 0, resistance: float = 0) -> float:
+
+    """
+    This function calculates the any one
+    of the three ohms_law values voltage,
+    current, resistance, of electronics.
+    >>> ohms_law(voltage=10, resistance=5,current=0 )
+    2.0
+    >>> ohms_law(current=1, resistance=10, voltage=0)
+    10.0
```

Comment:
Test for numbers with fractional parts (1.5) and negative numbers, etc.

Reply:
No problem i will be right back